setup.sh

```diff
@@ -18,16 +19,16 @@ declare -r app_name='kw'
 ## Following are the install paths
 ##
 # Paths used during the installation process
-declare -r kwbinpath="$HOME/.local/bin/$app_name"
-declare -r binpath="$HOME/.local/bin"
-declare -r libdir="$HOME/.local/lib/$app_name"
-declare -r sharedir="${XDG_DATA_HOME:-"$HOME/.local/share"}/$app_name"
-declare -r docdir="$sharedir/doc"
-declare -r mandir="$sharedir/man"
-declare -r sounddir="$sharedir/sound"
-declare -r databasedir="$sharedir/database"
+declare kwbinpath="${HOME}/.local/bin/$app_name"
+declare binpath="${HOME}/.local/bin"
+declare libdir="${HOME}/.local/lib/$app_name"
+declare sharedir="${XDG_DATA_HOME:-"$HOME/.local/share"}/$app_name"
+declare docdir="${sharedir}/doc"
+declare mandir="${sharedir}/man"
+declare sounddir="${sharedir}/sound"
+declare databasedir="${sharedir}/database"
```

Comment:
Consider making another commit just to use the upper case for all of these constant variables since it is more aligned with the overall code style.

Reply:
ok